JS/cart.js

```diff
@@ -254,7 +283,23 @@ function createCart2(article) {
     cart_div.appendChild(grid_article_in_cart);
 
     let line = document.createElement('div');
-    line.style = "padding : 10px;";
+    line.className = "line_in_cart";
     cart_div.appendChild(line);
-    return cart_div
-}
\ No newline at end of file
+    return cart_div;
+}
+/**
+ * Function used when we change the quantity of an article to save the changes into the sessionStorage
+ * @param {any} article : object that contains the article
+ * @param {any} elem : the input that contains the value of the quantity chosen for an article
+ */
+function update_json_file(article, elem) {
+    var old_data_saved = JSON.parse(sessionStorage.getItem('articleToCart2'));
+    var tab = [];
+    for (i = 0; i < old_data_saved.length; i++) {
+        if (old_data_saved[i].id == article.id) {
+            old_data_saved[i].quantity = elem.value;
+        }
+        tab.push(old_data_saved[i]);
```

Comment:
there is no need to use an additional array here as you are making changes in place.

Reply:
Effectively. I don't know why I did that anymore but it's corrected.